.redhat-ci.Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN dnf install -y \
         python-dbus \
         pylint \
         python-slip-dbus \
-        python-docker-py \
+        python2-docker \
```

Comment:
Yes I think we should wait for F26 to make this change.  I don't believe 3.5 vs 3.6 should be an issue.